tests/integration/test_render.py

```diff
@@ -8,22 +25,73 @@ class TestDragAndDropRender(BaseIntegrationTest):
     PAGE_TITLE = 'Drag and Drop v2'
     PAGE_ID = 'drag_and_drop_v2'
 
-    def setUp(self):
-        super(TestDragAndDropRender, self).setUp()
-
-        scenario_xml = "<vertical_demo><drag-and-drop-v2/></vertical_demo>"
+    def load_scenario(self, item_background_color="", item_text_color=""):
+        scenario_xml = """
+<vertical_demo>
+    <drag-and-drop-v2 item_background_color='{item_background_color}' item_text_color='{item_text_color}' />
+</vertical_demo>
+        """.format(item_background_color=item_background_color, item_text_color=item_text_color)
         self._add_scenario(self.PAGE_ID, self.PAGE_TITLE, scenario_xml)
 
         self.browser.get(self.live_server_url)
         self._page = self.go_to_page(self.PAGE_TITLE)
 
-    def _test_style(self, item, style_settings):
-        style = item.get_attribute('style')
+    def _test_style(self, element, style_settings, element_type):
+        style = element.get_attribute('style')
         for style_prop, expected_value in style_settings.items():
+            if style_prop == 'color' or style_prop == 'background-color' and expected_value.startswith('#'):
+                expected_value = Colors.rgb(expected_value)
             expected = u"{0}: {1}".format(style_prop, expected_value)
             self.assertIn(expected, style)
+        if element_type == "item":
+            self._test_item_style(element, style_settings, style)
```

Comment:
@bradenmacdonald I added logic for checking the color that's actually used in the browser. Leaving the logic for checking the `style` string in place though, since I think it makes sense to check background and text colors from two angles (i.e., to check both what users see in the browser and whether the mechanism for injecting custom styles is working).